salt/proxy/cimc.py

```diff
@@ -179,11 +187,18 @@ def get_config_resolver_class(cid=None, hierarchical=False):
                                 decode=True,
                                 verify_ssl=False,
                                 raise_error=True,
+                                status=True,
                                 headers=DETAILS['headers'])
 
+    if str(r['status']) not in ['200', '201', '202', '204']:
```

Comment:
This is identical to L156. Refactor for DRY?

Reply:
I'll change this to be included in a single function.